Makefile

```diff
@@ -27,11 +26,12 @@ in/countries.dat:
 in/continents.geojson: continents.geojson.gz
 	gunzip -c $< > in/continents.geojson
 
-tmp/$(UNLOCODEFILE): | tmp
-	wget -c https://service.unece.org/trade/locode/$(UNLOCODEFILE) -O $@
-
-unlocode: tmp/$(UNLOCODEFILE) | in
-	unzip -u tmp/$(UNLOCODEFILE) -d in/
+unlocode: | in
+	wget -c https://raw.githubusercontent.com/datasets/un-locode/${UNLOCODEREVIOSION}/data/subdivision-codes.csv -O in/SubdivisionCodes.tmp.csv
+	wget -c https://raw.githubusercontent.com/datasets/un-locode/${UNLOCODEREVIOSION}/data/code-list.csv -O in/CodeList.tmp.csv
+	awk 'NR > 1' in/SubdivisionCodes.tmp.csv > in/SubdivisionCodes.csv
+	awk 'NR > 1' in/CodeList.tmp.csv > in/CodeList.csv
+	rm in/*.tmp.csv
```

Comment:
This is not a proper `Makefile`, it's a script (see deleted lines for how it can work in a better way with dependencies).

Reply:
Split into files targets. Is that what you needed?

Reply:
Much better this way. I'd still keep downloaded files, but this works too.